statesync/syncer.go

```diff
@@ -504,6 +504,6 @@ func (s *syncer) verifyApp(snapshot *snapshot, appVersion uint64) error {
 		return errVerifyFailed
 	}
 
-	s.logger.Info("Verified ABCI app", "height", snapshot.Height, "appHash", snapshot.trustedAppHash)
+	s.logger.Info("Verified ABCI app", "height", snapshot.Height, "appHash", log.NewLazySprintf("%X", snapshot.trustedAppHash))
```

Comment:
Thanks for this.
Would you mind replacing line 494 by:

```go
"expected", fmt.Sprintf("%X", snapshot.trustedAppHash),
```
(no need for lazy formatter as it is an error condition, not a common path)

Reply:
updated!

Reply:
Thanks!